flow/envs/base_env.py

```diff
@@ -9,9 +9,12 @@
 import traceback
 import numpy as np
 import random
+from flow.renderer.pyglet_renderer import PygletRenderer as Renderer
 
 import traci
 from traci import constants as tc
+VAR_TIME = 102
```

Comment:
Magic numbers should at the least have comments or be imported from traci_constants if possible

Reply:
Solved.